backend/src/main/resources/application-dev.yml

```diff
@@ -12,12 +7,28 @@ spring:
       port: 27017
       authentication-database: admin
 
-security:
-  oauth2:
-    client:
-      client-id: xbdd
-      client-secret: secret
-      accessTokenUri: http://localhost:8080/oauth/token
+  security:
+    oauth2:
+      client:
+        registration:
+          github:
+            client-id: your-github-app-client-id
```

Comment:
We will need to update our deploy scripts for this change to work on ng as the string matching will be different